pydra/engine/tests/test_shelltask.py

```diff
@@ -2267,12 +2267,12 @@ def change_name(file):
     )
 
     # TODO: not sure why this has to be string
-    in_file = Path(os.path.dirname(os.path.abspath(__file__))) / "data" / "foo.nii"
+    in_file = Path(__file__).parent / "data_tests" / "test.nii.gz"
 
     # separate command into exec + args
     shelly = ShellCommandTask(
         name="bet_task", executable="bet", in_file=in_file, input_spec=bet_input_spec
     )
     assert shelly.inputs.executable == "bet"
     assert shelly.cmdline == f"bet {in_file} {in_file}_brain"
-    # res = shelly(plugin="cf")
+    res = shelly(plugin="cf")
```

Comment:
since we don't really want to run bet in pydra, should we just remove the actual execution?

Reply:
we actually don't run this test... see the decorator